graftm/clusterer.py

```diff
@@ -47,7 +48,9 @@ def uncluster_annotations(self, input_annotations, reverse_pipe):
                 placed_alignment_base = placed_alignment_file.replace('_clustered.fa', '')
             output_annotations[placed_alignment_base] = {}
             for rep_read_name, rep_read_taxonomy in cluster_classifications.iteritems():
-
+                slashendingregex = self.slash_ending_regex.match(rep_read_name)
```

Comment:
matching it in several places seems quite repetitive, might be best to detect it once and then set a variable in the unpack_sequences object, and then refer to that? Not 100% sure on this.